src/main/kotlin/com/cognifide/gradle/common/http/HttpClient.kt

```diff
@@ -127,7 +134,7 @@ open class HttpClient(private val common: CommonExtension) {
         useDefaults()
     }
 
-    fun HttpClientBuilder.useDefaults() {
+    private fun HttpClientBuilder.useDefaults() {
```

Comment:
this was intentionally public as it is used in some lamba and that lambda could be overidden then reuse `useDefaults()` and add own logic afterwards